docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -1,88 +1,134 @@
 ---
-title: PROXY protocol
-description: How to securely configure PROXY protocol usage with Teleport
+title: Run Teleport Behind a Layer 4 Load Balancer
+description: Describes how to securely configure Teleport to use the PROXY protocol so you can run the Auth Service and Proxy Service behind a layer 4 load balancer.
 ---
-<Admonition type="note" scope={["cloud", "team"]}>
-Users of Teleport Cloud don't need to manage PROXY protocol setting manually. It's set to `on` automatically 
-and can't be changed, since cloud Proxy/Auth services managed by Teleport run behind a L4 load balancer with PROXY protocol configured. 
-</Admonition>
 
-## What is PROXY protocol
+This guide shows you how to configure the Teleport Auth Service and Proxy
+Service to run behind a layer 4 (l4) load balancer. 
 
-The PROXY protocol is a prefix added to the TCP connection containing information about the client IP. 
-It is most commonly used when networking setup includes a Layer 4 (L4) load balancer between the user and the endpoint service,
-like Teleport Proxy/Auth.
-L4 load balancers, by design, do not retain the original client's IP address and port when forwarding the connection and 
-PROXY protocol allows to overcome this problem by adding the client's original IP address and port before the actual TCP stream.
+In this setup, the Auth Service and Proxy Service rely on the PROXY protocol to
+retrieve the IP addresses of clients when behind an L4 load balancer. Having
+reliable client IP information is important from a security standpoint,
+because features like audit logging and IP pinning depend on it. If the PROXY
+protocol is not configured correctly, these features will be compromised.
 
-An example of the PROXYv1 protocol header:
+<Notice type="note">
+
+Users of Teleport Enterprise (Cloud) do not need to manage PROXY protocol
+setting. Teleport-managed Auth Service and Proxy Service deployments run behind
+an L4 load balancer with the PROXY protocol configured. 
```

Comment:
@sclevine is this technically accurate?

Reply:
We have two layers of load balancing:
- AWS NLBs, which are non-terminating L4 load balancers
- Envoy, which is a connection (but not TLS) terminating load balancer (harder to categorize in OSI)

We do use PROXY protocol.

Reply:
Thanks for confirming :+1: